File: api/DTOs/Finance/UpdatePaymentConfirmationDto.cs
namespace api.DTOs.Finance
{
    public class UpdatePaymentConfirmationDto
    {
        public int VoucherEntryId { get; set; }
        public bool DrEntryApproved { get; set; }
        public DateTime DrEntryApprovedOn { get; set; }
        public int DrEntryApprovedByEmployeeById {get; set;}
    }
}

File: client/src/app/_dtos/finance/updatePaymentConfirmationDto.ts
export interface IUpdatePaymentConfirmationDto
{
	voucherEntryId: number;
	drEntryApproved: boolean;
	drEntryApprovedOn: Date;
	drEntryApprovedByEmployeeId: number;
}

export class UpdatePaymentConfirmationDto
{
	voucherEntryId= 0;
	drEntryApproved=false;
	drEntryApprovedOn= new Date();
	drEntryApprovedByEmployeeId=0;
}